packages/rspack/src/normalModule.ts

```diff
@@ -64,7 +64,9 @@ export class NormalModule {
 		let hooks = compilationHooksMap.get(compilation);
 		if (hooks === undefined) {
 			hooks = {
-				// loader: new SyncHook(["loaderContext", "module"]),
+				// TODO: figure out why tsc complain about this
+				// @ts-ignore
```

Comment:
If I comment out this `ts-ignore` tsc will complain about this: 
![image](https://user-images.githubusercontent.com/17974631/236752324-ee22006b-59cd-4579-97f5-1ff349a0134b.png)
I don't have too much time to dig into this issue. just workaround for now, because we use same code as webpack https://github.com/webpack/webpack/blob/9fcaa243573005d6fdece9a3f8d89a0e8b399613/lib/NormalModule.js#L234-L234

Reply:
move the loader hook here